lib/call.js

```diff
@@ -35,12 +35,13 @@ var call = function call(path, params, callback) {
   addParams["widget"] = props.widgId;
 
   var fhResp = {};
-  var req = https.request(options, function (res) {
+  var protocolLibrary = props.protocol === 'http' ? http : https;
```

Comment:
What about something like `protocolLibrary = require(props.protocol);`? Then you don't need to require both of them.